examples/typescript-decorators/README.md

```diff
@@ -0,0 +1,15 @@
+# TypeScript decorators example using `express-decorators` package
+
+Minimal [TypeScript](https://www.typescriptlang.org/) setup for tinyhttp using [ts-node](https://github.com/TypeStrong/ts-node).
```

Comment:
Could you please put in description that it uses decorators? ATM it looks the same as typescript example